src/X.Extensions.Serilog.Sinks.Telegram/Extensions/LogEventLevelExtensions.cs

```diff
@@ -0,0 +1,34 @@
+using JetBrains.Annotations;
+using Microsoft.Extensions.Logging;
+
+namespace X.Extensions.Serilog.Sinks.Telegram.Extensions;
+
+/// <summary>
+/// Helper methods for <see cref="LogEventLevel"/>.
+/// </summary>
+[PublicAPI]
+public static class LogEventLevelExtensions
+{
+    /// <summary>
+    /// Converts <see cref="LogEventLevel"/> to <see cref="LogLevel"/>.
+    /// </summary>
+    /// <param name="level">
+    /// LogEventLevel value.
+    /// </param>
+    /// <returns>
+    /// LogLevel value.
+    /// </returns>
+    public static LogLevel ToLogLevel(this LogEventLevel level)
```

Comment:
This produces redundant dependency on Microsoft.Extensions.Logging. I don't want to add dependencies just to keep the DB as unified as possible.
We can introduce a shared library and add a render interface there.


```cs
public interface ILogLevelMarkerRenderer
{
    string RenderMarker<TLevel>(TLevel level) where T : struct, Enum
} 
```

Then, each lib can have its implementation for that interface.

Another option is to create a set of internal models for LogLevel(enum), LogEvent, and so on, but in high-load scenarios, mapping will impact performance. Otherwise, we have Telegram bot API limits that prevent us from sending too many logs, so at least for now, performance shouldn't be hated so much.